Tests/GPU/libamrex_CUDA/GNUmakefile

```diff
@@ -1,4 +1,4 @@
-AMREX_INSTALL_DIR ?= ../../../tmp_install_dir
+AMREX_INSTALL_DIR = ../../../tmp_install_dir
```

Comment:
I don't think that this one and the three other instances of `AMREX_INSTALL_DIR` should be changed. If someone is building as a library, they might want to specify some other directory on the system to put the installation, maybe somewhere in `/home/...` or `/usr/...`. This would overwrite that choice.